packages/api/src/AwilixHelpers.ts

```diff
@@ -51,5 +51,13 @@ function makeProxyAwilixFriendly(
   injectable: (...args: any[]) => any,
   realFunction: (...args: any[]) => any
 ) {
-  injectable.toString = () => "function " + realFunction.toString();
+  injectable.toString = () => {
```

Comment:
Write actual tests for this. It's really important that it works right for literally the entire app. Test against `fn = () => . . .` class methods too.

Reply:
Open a ticket to do this?